src/client/Admin.js

```diff
@@ -66,14 +66,9 @@ var Admin = React.createClass({
         });
     },
     confirmOpenPrintPage() {
+        this.getServerStatus();
```

Comment:
My intention was for it to perform the request and verify that we're signed in.. It turned out to not really work..